app/build.gradle

```diff
@@ -47,7 +47,7 @@ android {
             dimension "server"
             applicationId = "${packageId}.local"
             ext {
-                url = "http://localhost:3000"
+                url = "http://10.0.2.2:3000"
```

Comment:
Is this something for your environment? I think localhost is working for other developers.